specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -0,0 +1,1674 @@
+              "type": "array",
+              "items": {
+                "$ref": "./CommonDefinitions.json#/definitions/Identifier"
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CustomHostnameSitesCollection": {
+      "description": "Collection of custom hostname sites",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomHostnameSites"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DeploymentLocations": {
+      "description": "List of available locations (regions or App Service Environments) for\ndeployment of App Service resources.",
+      "type": "object",
+      "properties": {
+        "locations": {
+          "description": "Available regions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeoRegion"
+          }
+        },
+        "hostingEnvironments": {
+          "description": "Available App Service Environments with full descriptions of the environments.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/AppServiceEnvironment"
+          },
+          "x-ms-identifiers": []
+        },
+        "hostingEnvironmentDeploymentInfos": {
+          "description": "Available App Service Environments with basic information.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HostingEnvironmentDeploymentInfo"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "AseRegion": {
+      "description": "ASE region.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ASE region resource specific properties",
+          "type": "object",
+          "properties": {
+            "displayName": {
+              "description": "Display name for region.",
+              "type": "string",
+              "readOnly": true
+            },
+            "standard": {
+              "description": "Is region standard.",
+              "type": "boolean",
+              "readOnly": true
+            },
+            "dedicatedHost": {
+              "description": "Dedicated host enabled.",
+              "type": "boolean",
+              "readOnly": true
+            },
+            "zoneRedundant": {
+              "description": "Zone redundant deployment enabled.",
+              "type": "boolean",
+              "readOnly": true
+            },
```

Comment:
non-blocking, consider use enum over boolean for easier expansion if needed in future